app/pdl.yaml

```diff
@@ -431,6 +471,26 @@ paths:
           description: Unauthorized
         500:
           description: Internal error
+  /pdl/titles/title-ii/programs/eqip-ira/category:
```

Comment:
It may be better to call the endpoint as /practices or /practice-names.

Practice categories are different entities.

Reply:
Changed to practice-names